msteams-platform/tutorials/get-started-dotnet-app-studio.md

```diff
@@ -67,10 +67,10 @@ After the repo is cloned, use Visual Studio to open the solution file **Microsof
 
 When the app starts, a browser window opens with the root of the app launched. You can go to the following URLs to verify that all the app URLs are loading:
 
-- `https://localhost:44327/`
-- `https://localhost:44327/hello`
-- `https://localhost:44327/first`
-- `https://localhost:44327/second`
+- `http://localhost:3978/`
```

Comment:
Could you please update this to https?

Reply:
Completed.

Reply:
@shaikhwajeed, please update the comment given by @Wajeed-msft